Dockerfile

```diff
@@ -59,11 +59,9 @@ COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
 # non-root
 USER 82
 
-COPY ./composer.json /var/www/html/composer.json
-RUN composer install --no-dev --no-scripts --no-autoloader
+COPY ./composer.json ./composer.lock /var/www/html/
```

Comment:
Methodology: 1. Add lock here because I don't think our codebase is in the best position for uniing w/o lock at the moment.